kotlin/change-basemaps/src/main/java/com/esri/arcgisruntime/sample/switchbasemaps/MainActivity.kt

```diff
@@ -0,0 +1,167 @@
+/* Copyright 2018 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgisruntime.sample.switchbasemaps
+
+import android.content.res.Configuration
+import android.os.Bundle
+import android.support.v7.app.ActionBarDrawerToggle
+import android.support.v7.app.AppCompatActivity
+import android.view.MenuItem
+import android.view.View
+import android.widget.AdapterView
+import android.widget.ArrayAdapter
+
+import com.esri.arcgisruntime.mapping.ArcGISMap
+import com.esri.arcgisruntime.mapping.Basemap
+import kotlinx.android.synthetic.main.activity_main.*
+
+class MainActivity : AppCompatActivity() {
+
+  private var mNavigationDrawerItemTitles: Array<String>? = null
```

Comment:
You can use `lateinit` to avoid the nullable type. You should only very rarely use `!!`.
```kotlin
private lateinit var mNavigationDrawerItemTitles: Array<String>
```

Reply:
Or better yet, initialize it immediately since it could definitely be a `val`.